src/representation_theory/group_cohomology/resolution.lean

```diff
@@ -233,6 +233,117 @@ module.free.of_basis (of_mul_action_basis k G n)
 
 end basis
 end group_cohomology.resolution
+namespace finsupp
+-- I am not sure where to put these
```

Comment:
`linear_algebra.finsupp` is probably reasonable; or next to `finsupp.lift` if the imports allow.

Reply:
I would have to add the import `algebra.algebra.basic`, but I can't think of any appropriate file where I wouldn't have to add imports.